lib/entities/personal-access-token.ts

```diff
@@ -5,14 +5,16 @@ import { wrapCollection } from '../common-utils'
 import { MetaSysProps, DefaultElements, MakeRequest } from '../common-types'
 
 export type PersonalAccessTokenProp = {
-  sys: MetaSysProps
+  sys: MetaSysProps & { expiresAt?: number }
```

Comment:
```suggestion
  sys: MetaSysProps & { expiresAt?: string }
```

Reply:
@Annmary12 I actually believe it should be a number here, right?
As we're sending the number to the API as well:

<img width="363" alt="Screenshot 2023-09-01 at 15 22 51" src="https://github.com/contentful/contentful-management.js/assets/3019206/86894b7f-00b7-42fb-a167-39fbe4f1aa32">

Reply:
Also, in `GenerateCMATokenDialog` on lines 148-150, we are also setting the following payload as number:

<img width="396" alt="Screenshot 2023-09-01 at 15 25 53" src="https://github.com/contentful/contentful-management.js/assets/3019206/2279c27a-bf2a-4ed9-bd24-581d8e971a2f">

Reply:
I was not referring to `expiresIn`, it is `expiresAt` that I highlighted which is a date string

Reply:
Sorry, you are correct, not sure why I went onto `expiresIn`. 🤦 
I'll see to fix this. 👍